cmake/std/atdm/load-env.sh

```diff
@@ -5,9 +5,8 @@
 ################################################################################
 
 # Assert this script is sourced, not run!
-called=$_
-if [ "$called" == "$0" ] ; then
-  echo "This script '$0' is being called.  Instead, it must be sourced!"
+if [ "${BASH_SOURCE[0]}" == "${0}" ] ; then
```

Comment:
That's a bash variable, no? So how would this work with other shells?

Reply:
> That's a bash variable, no? So how would this work with other shells?

@cgcgcg, Scot Swan says this fixed his use case.  I could not confirm it.  This script was never meant to be used by any shell other than bash but if it fixes that EMPIRE use case, so be it.

Can you approve this so it can be merged so EMPIRE can have this?